packages/config/src/api/site_info.ts

```diff
@@ -96,12 +123,7 @@ const getIntegrations = async function ({ api, ownerType, ownerId, testOpts }) {
   const baseUrl = host ? `http://${host}` : `https://api.netlifysdk.com`
```

Comment:
NIT: I would use the URL constructor for that

Reply:
lovely idea